src/main/resources/messages/RsBundle.properties

```diff
@@ -53,6 +53,8 @@ action.Rust.RsExtractStructFields.choose.fields.title=Choose Fields to Extract
 action.Rust.RsExtractStructFields.choose.name.dialog.title=Enter Name For the New Struct
 action.Rust.RsExtractStructFields.choose.name.dialog.invalid.name=Invalid struct name
 action.Rust.RsExtractStructFields.command.name=Extracting Struct Fields
+action.Rust.RsExtractTrait.text=Extract Trait
```

Comment:
Maybe use ellipsis `…` here, as in other refactoring names?
<img width="264" alt="Screenshot 2021-12-21 at 17 58 39" src="https://user-images.githubusercontent.com/4854600/146977688-4ad1d705-9c94-46cd-bcae-551c43ea108a.png">